test/remote/gateways/remote_credorax_test.rb

```diff
@@ -286,8 +306,7 @@ def test_failed_refund
   end
 
   def test_successful_referral_cft
-    options = @options.merge(@normalized_3ds_2_options)
-    response = @gateway.purchase(@amount, @three_ds_card, options)
+    response = @gateway.purchase(@amount, @credit_card, @options)
```

Comment:
this did not seem dependent on 3ds, and using the 3ds card doesn't result in a completed transaction now, so updated these to use the regular credit card. if my assumption is wrong, definitely let me know! cc @leila-alderman

Reply:
@britth I haven't actually tested this transaction type without sending 3DS. I went with the 3DS version based on Credorax's documentation for `referral_cft` transactions:

> For transactions made with a Mastercard payment card, the original transaction to which the CFT refers must include 3D secure data in order for the referral CFT transaction to be processed successfully.

I'm puzzled as to why these transactions would no longer be successful. 🤔

Reply:
interesting, thanks for that info! yeah, it's very weird - we can get to the point where 3ds is triggered and awaiting authorization, but it's a bit tricky to try to simulate an actual 3ds auth; but it's very interesting that the 3ds card was working frictionlessly before. i'll check with credorax to see if there's any updated test card data 👍

Reply:
aha, the test cards did change - updated the card number and changed these tests back to their original state